docs/source/create_channel/create_channel_participation.md

```diff
@@ -4,19 +4,21 @@ Fabric v2.3 introduces the ability to create channels between network members wi
 
 **How this process differs from the legacy process:**
 
- * **System channel no longer required** - Previously you had to define a system channel using `configtxgen` on which all application channels were based.
- * **Ordering service consortium no longer required** - You no longer need to define the set of organizations, also referred to as the "consortium", to transact on an application channel.
- * **Channel simplicity** - There is no longer the concept of a system channel or an application channel. There is simply a "channel" that allows two or more organizations to transact on the blockchain network. Each channel is administered separately, and each ordering service node can join or leave as needed.
+ * **System channel no longer required**: Previously a channel managed by the ordering service (called the "system channel") was defined using the `configtxgen` tool to create a genesis block. This channel was exclusively managed by the ordering service and was the template on which all application channels were based. Besides the creation of the system representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.
+ * **Consortium no longer required**: You no longer need to define the set of organizations, known as the "consortium", who are permitted to create channels on a particular ordering service. With this new process, **all channels are application channels**, so the concept of a list of organizations who can create channels no longer applies. Any set of organizations can get together and create a channel using a defined set of ordering nodes (which become the ordering service of that channel). Additionally, if the MSP of the peer organization, as listed in the system channel, needs to be changed, the peer organization to coordinate with an orderer organization to change the MSP.
```

Comment:
`Additionally, if the MSP of the peer organization, as listed in the system channel, needs to be changed, the peer organization to coordinate with an orderer organization to change the MSP.` This reads as halfway in between what I think you were going for. Do you want to highlight that you no longer need to coordinate with an orderer organization here?

Reply:
I think some words were accidentally dropped in there. I changed it to:
`Additionally, if the MSP of the peer organization, as listed in the system channel, needs to be changed, the peer organization no longer needs to to coordinate with an orderer organization to change the MSP.`